posit-dev/positron
# Upstream VSCode Python changes from v2024.14.1 to v2024.20.0

Addresses #5567.

I also bumped our kernel dependencies (jedi, jedi-language-server, etc) which required updates to the patch files.





### QA Notes


Build and run Positron locally and check that things generally look okay. I'll also run the full test suite GH action.
